docs/source/swift-scripting.md

```diff
@@ -1,30 +1,29 @@
 ---
-title: Swift Scripting
+title: Swift scripting
+sidebar_title: Swift scripting (beta)
```

Comment:
ooh neat, I didn't know you could do that.

Reply:
It's pretty new! :D